src/components/Data/Browse/IndexBranch.vue

```diff
@@ -0,0 +1,165 @@
+<template>
+  <div v-bind:class="{ 'open': open }">
+    <i v-if="collectionCount(index)" class="fa fa-caret-right tree-toggle" aria-hidden="true" @click="toggleBranch()"></i>
+    <!-- v-link="{name: 'DataIndex', params: {index: index.name}}" -->
+    <a class="tree-item truncate"
+       v-bind:class="{ 'active': isIndexActive($route, index.name) }">
+      <i class="fa fa-database" aria-hidden="true"></i>
+      <strong>{{index.name}}</strong> ({{collectionCount(index)}})
+    </a>
+    <ul class="collections">
+      <li v-for="collection in index.collections.stored">
```

Comment:
Hum, why we have stored collection in top of realtime ones ? 
I think we have to compute a list containing all collection, that permit to apply a sort order.